src/front/src/pages/Error.tsx

```diff
@@ -1,7 +1,35 @@
+import * as React from 'react';
+import {Grid} from "@mui/material";
+import Box from "@mui/material/Box";
 
 function Error() {
 
-    return (<h1>Error</h1>);
+    return (
+        <Box>
+            <Grid alignContent={"center"}>
```

Comment:
Is this CSS directly in a tag ? I think you should use CSS classes ?

Reply:
This isn't possible with this configuration of React server, because we don't take styles folder for the build